src/encoded/static/components/motifs.js

```diff
@@ -125,7 +125,7 @@ export const Motifs = (props) => {
     const urlBase = props.urlBase;
 
     // Filter results to find ones with associated PWM data
-    const pwmLinkList = results.filter(d => d.documents[0] && d.documents[0].document_type === 'position weight matrix');
+    const pwmLinkList = results.filter(d => d.documents && d.documents[0] && d.documents[0].document_type === 'position weight matrix');
```

Comment:
Is this right enough? Just look at the first document (which is right for now)?

Reply:
I guess this will fail if more than 1 doc is added to these annotation files.   But I don't really want to fabricate a test case for it.   Also there is a (bad) case where there are >1 documents with type = 'position weight matrix' (this shouldn't happen on purpose)   @emro how would you write this to pull all the documents with this type?  My JS is rusty.